Makefile

```diff
@@ -34,6 +34,10 @@ requirements: venv
 	$(PIP) freeze > requirements.txt
 	@$(MAKE) $(MAKE_TAG) echo-green msg="\n$@.txt updated! ✨\n"
 
+question-schema:
```

Comment:
1. Why don't we make a target schemas that generates all the json-schemas ?
2. Are you sure this is working ? I mean if you have tjs globally then it works for sure. But how will the command tjs be resolved if you don't have it globally.!? In addition, you are installing it in the frontend dir while the make target is in the top level dir.!

Reply:
1. Can you give me an example ?
2. The `tjs` module seems to be found in `./node_modules` by the command. As the `node_modules` dir is also in the top level dir, it works 
<img src="https://user-images.githubusercontent.com/64146788/203335960-4f754bc0-d3d1-4852-bd11-d424de8cff5c.png" width="40%">

Reply:
Where does your `node_modules` lives ? Is it in `quizoot/` or `quizoot/frontend` ?

Reply:
If you use the `tjs` command in `quizoot/`, I believe you won't be able to use the right binary. `node_modules` resolves from the bottom to the top. Not the other way.

Reply:
I suggest we merge it like that, after what you'll try to run the command yourself and see the behavior.
If it goes wrong, I will make the required changes 🤔 
As you said I'm propbably using a wrong binary.

Reply:
I can try locally by switching to your branch. let me do that first

Reply:
I just tested by :
- removing `typescript-json-schema` globally
- moving to `frontend` dir and installing the packages
- running `make question-schema`

And i got this:
<img width="429" alt="Screenshot 2022-11-23 at 13 14 12" src="https://user-images.githubusercontent.com/58826859/203544297-7b6f484c-7896-4718-ab6a-3e198b51aeb9.png">


Then I reinstalled `typescript-json-schema` globally and the make target was working again. It proves that you're using the global binary...

Reply:
Proposed solutions:
-------------------
1. Create a `package.json` in the top level dir, add `typescript-json-schema` in devDependencies. Doing so, we'll use the right binary
2. Use the binary in `frontend/node_modules/typescript-json-schema/....` in your make target. (I don't like this option)
3. Create a `package.json` in `schemas/` and maybe write a js script that import it and generate the right thing. You can then invoke this script in your make target


What do you prefer or suggest ?

Reply:
I implemented 1 and 3.

* It is easy with method 1, but I don't really like having a `package.json` in the top level dir.
* Method 3 looked great, but I end up stucked on getting the generated schema data in a file.

I pushed both, expecting you to fix method 3. Otherwise I'll delete it and keep method 1.

What do you think about it ?